packages/contentstack-seed/package.json

```diff
@@ -1,14 +1,13 @@
 {
   "name": "@contentstack/cli-cm-seed",
   "description": "create a Stack from existing content types, entries, assets, etc.",
-  "version": "1.5.0",
+  "version": "1.5.1",
```

Comment:
@vkalta package-lock is not updated with new bumped version.
Can you push the updated package-lock file?